Imperium - Space Marines.cat

```diff
@@ -90,7 +90,6 @@
     <categoryEntry id="551d-174d-e214-52d6" name="Land Raider Redeemer" hidden="false"/>
     <categoryEntry id="6f4f-95b8-5802-55b6" name="Repulsor" hidden="false"/>
     <categoryEntry id="cca9-5590-14a6-19e8" name="Repulsor Executioner" hidden="false"/>
-    <categoryEntry id="fcd4-51e2-7b33-44cc" name="Repulsor Executioner" hidden="false"/>
```

Comment:
Scanned the file for this id and no other references came up, where as the above Rep Ex (cca9-5590-14a6-19e8) was referenced elsewhere.